recipes/loguru/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+
+required_conan_version = ">=1.54.0"
+
+
+def _int(flag):
+    if flag is not None and flag:
+        return "1"
+    else:
+        return "0"
+
+
+class LoguruConan(ConanFile):
+    name = "loguru"
+    version = "2.1.0"
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = 'CMakeLists.txt'
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/emilk/loguru"
+    license = "Unlicense"  # Public domain
+    description = "Loguru is a C++11 logging library."
+
+    options = {
+        "use_fmtlib": [True, False],
+        "shared": [True, False],
+        "catch_sigabrt": [True, False],
+        "verbose_scope_endings": [True, False],
+        "redefine_assert": [True, False],
+        "with_streams": [True, False],
+        "with_fileabs": [True, False],
+        "with_rtti": [True, False],
+        "replace_glog": [True, False],
+
+        "scope_text_size": "ANY",       # int
+        "filename_width": "ANY",        # int
+        "threadname_width": "ANY",      # int
+        "scope_time_precision": "ANY",  # int
+    }
+
+    default_options = {
+        "use_fmtlib": False,
+        "shared": False,
+        "catch_sigabrt": True,
+        "verbose_scope_endings": True,
+        "redefine_assert": False,
+        "with_streams": False,
+        "with_fileabs": False,
+        "with_rtti": True,
+        "replace_glog": False,
+
+        "scope_text_size": None,         # default: 196
+        "filename_width": None,          # default: 23
+        "threadname_width": None,        # default: 16
+        "scope_time_precision": None,    # 3=ms, 6=us, 9=ns, default: 3
+    }
+
+    def build_requirements(self):
+        self.tool_requires("cmake/3.25.0")
+
```

Comment:
```suggestion
```
it does not need cmake 3.25 for a very simple library.

Reply:
Do you mean I should remove the `cmake` build requirement altogether? Or to downgrade its version?

I mean, it currently uses CMake to build itself. Can I still remove cmake build dependency?